src/components/UserSelector.tsx

```diff
@@ -19,22 +61,23 @@ export const UserSelector: React.FC = () => {
       </div>
 
       <div className="dropdown-menu" id="dropdown-menu" role="menu">
-        <div className="dropdown-content">
-          <a href="#user-1" className="dropdown-item">
-            Leanne Graham
-          </a>
-          <a href="#user-2" className="dropdown-item is-active">
-            Ervin Howell
-          </a>
-          <a href="#user-3" className="dropdown-item">
-            Clementine Bauch
-          </a>
-          <a href="#user-4" className="dropdown-item">
-            Patricia Lebsack
-          </a>
-          <a href="#user-5" className="dropdown-item">
-            Chelsey Dietrich
-          </a>
+        <div
+          className={classNames('dropdown-content', {
+            'is-active': isPressed,
+          })}
+        >
+          {users.map(user => (
+            <a
+              href={`#user-${user.id}`}
+              className={classNames('dropdown-item', {
+                'is-active': userId === user.id,
+              })}
+              key={user.id}
+              onClick={() => handleClick(user)}
+            >
+              {user.name}
```

Comment:
Destructure `user`

Reply:
it`s bad idea, becouse i alredy have the variables NAME and USERID. So i can`t name not id and name, and can`t name userId and userName